docs/admin/node-problem.md

```diff
@@ -0,0 +1,238 @@
+
+```shell
+kubectl create -f node-problem-detector.yaml
+```
+
+### Addon Pod
+
+This is for those who have their own cluster bootstrap solution, and don't need
+to overwrite the default configuration. They could leverage the addon pod to
+further automate the deployment.
+
+Just create `node-problem-detector.yaml`, and put it under the addon pods directory
+`/etc/kubernetes/addons/node-problem-detector` on master node.
+
+## Overwrite the Configuration
+
+The [default configuration](https://github.com/kubernetes/node-problem-detector/tree/v0.1/config)
+is embedded when building the docker image of node problem detector.
+
+However, you can use [ConfigMap](/docs/user-guide/configmap/) to overwrite it
+following the steps:
+
+* **Step 1:** Change the config files in `config/`.
+* **Step 2:** Create the ConfigMap `node-problem-detector-config` with `kubectl  create configmap
+node-problem-detector-config --from-file=config/`.
+* **Step 3:** Change the `node-problem-detector.yaml` to use the ConfigMap:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: node-problem-detector-v0.1
+  namespace: kube-system
+  labels:
+    k8s-app: node-problem-detector
+    version: v0.1
+    kubernetes.io/cluster-service: "true"
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: node-problem-detector
+        version: v0.1
+        kubernetes.io/cluster-service: "true"
+    spec:
+      hostNetwork: true
+      containers:
+      - name: node-problem-detector
+        image: gcr.io/google_containers/node-problem-detector:v0.1
+        securityContext:
+          privileged: true
+        resources:
+          limits:
+            cpu: "200m"
+            memory: "100Mi"
+          requests:
+            cpu: "20m"
+            memory: "20Mi"
+        volumeMounts:
+        - name: log
+          mountPath: /log
+          readOnly: true
+        - name: config # Overwrite the config/ directory with ConfigMap volume
+          mountPath: /config
+          readOnly: true
+      volumes:
+      - name: log
+        hostPath:
+          path: /var/log/
+      - name: config # Define ConfigMap volume
+        configMap:
+          name: node-problem-detector-config
+```
+* **Step 4:** Re-create the node problem detector with the new yaml file:
+
+```shell
+kubectl delete -f node-problem-detector.yaml # If you have a node-problem-detector running
+kubectl create -f node-problem-detector.yaml
+```
+***Notice that this approach only applies to node problem detector started with `kubectl`.***
+
+For node problem detector running as cluster addon, because addon manager doesn't support
+ConfigMap, configuration overwriting is not supported now.
+
+## Kernel Monitor
+
+*Kernel Monitor* is a problem daemon in node problem detector. It monitors kernel log
+and detects known kernel issues following predefined rules.
+
+***Notice that Kernel Monitor only supports file based kernel log now, It doesn't support
```

Comment:
I would put this in a section immediately after the overview and before GCE titled ## Limitations